expressEngine/server.js

```diff
@@ -15,29 +17,30 @@ import seed from './db/seedData';
 
 // Set up the express app
 const app = express();
-
+config();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-app.use('/health-check', (req, res, next) => {
+app.use('/health-check', (req, res) => {
```

Comment:
@aman-successive You can create a route helper that can handle this particular route.

Reply:
Sir, I was just fixing some eslint errors because I was making changes in Server.js, this issue doesn't concern with routing.
I will create another issue for that and make the changes